### ISSUE TITLE
Allow using minute, hour and day as time unit in plots and experiment setup

### ISSUE THREAD

**Issue by jelgerjansen** [OpenModelica/OpenModelica#11382]
For building energy simulations we usually perform simulations over multiple days (or even a full year).
It would be convenient to change the unit of the time axis to minutes/hours/days if desired (instead of getting results in function of time in ks/Ms).
This feature would also be useful in the experiment setup window (e.g. to set a Start Time and Stop Time in days).

**Comment by bilderbuchi** [comment]
> It would be convenient to change the unit of the time axis to minutes/hours/days if desired (instead of getting results in function of time in ks/Ms).

An opt-in method or extension of the set of admissible prefixes sounds reasonable, but not a general removal of SI-prefixes for time, IMO -- e.g. `ms` are still a quite useful prefix in some simulation contexts.

Leaving aside the issue of sidereal vs. solar day (and year) length differences, how do you propose to deal with leap years? I.e. is a year 365 or 365.2425 days when converting? 
Life is much easier when you stay in the SI system 😅

**Comment by jelgerjansen** [comment]
I agree that the use of SI units remains the default, changing the time unit is rather an extension in the front-end (as it is in Dymola): setting a Start Time of e.g. 30 days in the setup window results in a Start Time of 2.592e6 seconds. The latter is 'internally' used by OM.
I would stick to minutes, hours, and days (and use a conversion factor of 60, 3600, and 86400). If you do a simulation over multiple years, the results are then displayed in days and its up to the user to deal with leap years.

**Comment by casella** [comment]
> I would stick to minutes, hours, and days (and use a conversion factor of 60, 3600, and 86400). If you do a simulation over multiple years, the results are then displayed in days and its up to the user to deal with leap years (in e.g. their used weather files).

I agree to that.

**Comment by casella** [comment]
@adeas31 please consider this for your to-do list

**Comment by casella** [comment]
It's done already, you can set the unit for the time axis here:

<img width="694" height="440" alt="Image" src="https://github.com/user-attachments/assets/416251b7-3307-4449-9b34-24801951fc38" />

**Comment by casella** [comment]
The idea is that the units of the plotting can be managed from the Variables browser. Both for each individual variable, by changing the displayUnit, and for the time variable.
